doc/content/examples/divertor_monoblock/index.md

```diff
@@ -0,0 +1,257 @@
+
+
+### Boundary conditions and history
+
+For the mass conservation equation, we use `FunctionNeumannBC` and `DirichletBC` to solve the 2$^{nd}$ term of [eq:concentration_weak].
+`FunctionNeumannBC` is used to treat the time-dependent plasma exposure at the top (plasma-exposed) surface,
+and `DirichletBC` is used to set the BC of the solute T atom concentration to zero at the inner CuCrZr tube (at $r = 6$ mm).
+For the energy conservation equation, we use `FunctionNeumannBC` and `DirichletBC` to solve the 2$^{nd}$ term of [eq:temperature_weak].
+`FunctionNeumannBC` is used to treat the time-dependent heat flux at the top (plasma-exposed) surface,
+and `DirichletBC` is used to treat the temperature increase in the cooling tube.
+`FunctionNeumannBC,` `FunctionDirichletBC,` and `DirichletBC` are MOOSE BCs commonly used to determine the BCs of the variables to be solved.
+
+We simulate a 1,600 s-cycle plasma discharge, with each cycle consisting of a 100 s plasma ramp-up,
+a 400 s steady-state plasma discharge, a 100 s plasma ramp-down, and a 1,000 s waiting phase.
+Up to 50 cycles are simulated to achieve the total 20,000 s steady-state plasma discharge.
+[fig:tritium_temperature_history] shows the integrated (solute, total and trapped) tritium concentration in the monoblock.
+It shows that implantation fluxes and temperatures varies linearly up to (from) their steady-state values from (up to) their initial values during ramp-up (ramp-down).
+During the steady-state plasma discharge, we set a heat flux of 10 MW/m$^{2}$ at the top of the 2D monoblock (at $y = 14.0$ mm)
+and a cooling temperature of 552 K at the inner CuCrZr tube (at $r = 6.0$ mm).
+We assume 100% T plasma with a 5.0 $\times$ 10$^{23}$ m$^{-2}$$\cdot$s$^{-1}$ plasma particle flux
+(which is 50% of 1.0 $\times$ 10$^{24}$ DT m$^{-2}$$\cdot$s$^{-1}$ plasma particle flux)
+and only 0.1% of the incident plasma particle flux (5.0 $\times$ 10$^{20}$ m$^{-2}$$\cdot$s$^{-1}$) entered the first layer of mesh at $y = 14.0$ mm.
+We treated this plasma exposure by setting the flux BC (5.0 $\times$ 10$^{16}$ m$^{-3}$$\cdot$s$^{-1}$) of the solute T atom concentration in the first layer of mesh
+(1.0 $\times$ 10$^{–4}$ m) at $y = 14.0$ mm in order to avoid simulating the complex plasma implantation and recombination phenomena
+via an extremely fine mesh size (∼10$^{–10}$ m) and a long computation time.
+The solute T atom concentration is set to zero at the inner CuCrZr tube (at $r = 6.0$ mm).
+
+!media examples/figures/divertor_monoblock_history.png
+  id=fig:tritium_temperature_history
+  caption=Temperature profiles (orange) and integrated tritium concentration profiles (blue) during two 1,600-second-cycle plasma discharges. This corresponds to Fig. 2 in Ref. [!cite](Shimada2024114438).
+  style=display:block;margin-left:auto;margin-right:auto;width:60%
+
+!listing test/tests/divertor_monoblock/divertor_monoblock.i link=false block=BCs
```

Comment:
Should the functions used be discussed as their own section since they are common in these BC objects? They are common for the material properties too.

Reply:
I added a section for the functions